### ISSUE TITLE
[bookshelf-duetai-bigquery] 

### ISSUE THREAD

**Issue by NabheetCloud** [googlecodelabs/feedback#1361]
Please correct the following

- mention about enabling Vertex AI API
- Step 10 gcloud functions call at the end is not correct. it should have --

**Comment by AbiramiSukumaran** [comment]
Yes. These changes are made in the codelab. Waiting to be published!
